src/preferences/dialog/dlgprefcolorsdlg.ui

```diff
@@ -62,7 +76,7 @@
          <string>Automatically assigns a predefined color to a newly created hotcue point, based on its index.</string>
         </property>
         <property name="text">
-         <string>Assign predefined colors to newly created hotcue points</string>
+         <string>Select consecutive palette colors for new hotcues</string>
```

Comment:
The palette colors aren't necessarily consecutive as they can be in any order depending on the "Assign to Hotcue Index" column. However, I'd suggest we remove this option completely - see PR #2578,

Reply:
I think it is very usable to have this option separate from the palette editor. 
Many users probably fine with the Mixxx colors and only like to select the default color or the color by number option. 
So I actually want the combobox back where we can select one or all colors from the current palette. 
I think you are used to the color by number feature, right? 
I like to have one default color. Currently the later is hidden and therefore unpredictable. 
We need to fix that as well.
Would it OK for you to bring that box back?